build.gradle.kts

```diff
@@ -6,7 +6,7 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
 
 plugins {
-    `kotlin-dsl`
```

Comment:
https://github.com/gradle-nexus/publish-plugin/pull/295#issuecomment-1893712584 raises a good point, where is java-gradle-plugin coming from, or we don't need it.

Reply:
Hmm, we use `com.gradle.plugin-publish` which provided `gradlePlugin {}`. We have publications set manually. They could be [some other](https://docs.gradle.org/current/userguide/java_gradle_plugin.html) which could be missing/misconfigured once the plugin is released without java-gradle-plugin.
We could add it - just in case ;-) - or release rc-2 hoping to have it discovered within a week or so.

Reply:
Never mind. From the [documentation](https://docs.gradle.org/current/userguide/java_gradle_plugin.html#plugin_publish_plugin):


```
[Plugin Publish Plugin](https://docs.gradle.org/current/userguide/java_gradle_plugin.html#plugin_publish_plugin)

Starting from [version 1.0.0, the Plugin Publish Plugin](https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.0.0) always auto-applies the Java Gradle Plugin (java-gradle-plugin) and the Maven Publish Plugin (maven-publish).
```

Reply:
Ah, of course...